### DISCUSSION TITLE
Add option to disable notifications pausing auto-hide

### DISCUSSION THREAD

**Discussion by johnch18** [valinet/ExplorerPatcher#2302] [Ideas]
I get a fair number of notifications from various applications, and they seem to prevent the taskbar from hiding. I'd like it if I could disable this behavior, so that the notifications display and briefly show the taskbar for a few seconds, after which it would hide itself. It would also be nice to be able to configure the time that the taskbar stays visible

**Comment by pyrates999** [comment]
You can do this by silencing notifications: https://support.microsoft.com/en-us/windows/turn-off-notifications-in-windows-during-certain-times-81ed1b25-809b-741d-549c-7696474d15d3
